envs.json

```diff
@@ -310,11 +310,11 @@
   },
   "tracer_ci": {
     "evm_loader": "eeLSJgWzzxrqKv1UxtRVVH8FX3qCQWUs9QuAjJpETGU",
-    "proxy_url": "https://ch2-graph.neontest.xyz/solana",
+    "proxy_url": "https://devnet.neonevm.org/",
     "network_ids": {
       "neon": 245022926
     },
-    "solana_url": "https://api.devnet.solana.com",
+    "solana_url": "http://neon-solrpc-devnet-lb-1.solana.p2p.org/0LcGBYwNa9b0ifu7odnVPEF9TiYHQUXfOuxrTXlH",
```

Comment:
Is it okay to use p2p node?
I heard that we had a plan to move from them.

Reply:
@andreisilviudragnea we can't use p2p node (we disabled it). We have our devnet solana nodes, but we can't share it in public! We have logic that in devnet we are using SOLANA_URL from environment, you can use it here too https://github.com/neonlabsorg/neon-tests/blob/develop/conftest.py#L85

Reply:
@gigimon Done